clusterloader2/testing/experimental/storage/pod-startup/max_volumes_per_node/25_pods_per_node/override.yaml

```diff
@@ -0,0 +1,3 @@
+PODS_PER_NODE: 25
+VOLUMES_PER_POD: 1
+NODES_PER_NAMESPACE: 1
```

Comment:
I couldn't comment on the file below - so let me comment here.
After looking into that, I don't think we should commit this file into this repo.
In fact, I just realized that it only contains data that we already have and can actually be viewed in perf-dash.k8s.io
[well, something is broken as of now it doesn't show results since uyesterday morning, but I'm pretty sure it's minor issue]

So I think we should just get rid of that json file.

Reply:
So I think I got it wrong your and @msau42 comment about keeping initial data.

I understand it as keeping test results, yes this data can be seen from dashboard but I thought it only shows some windowed data, and at the future we cant see initial results.

So could you give me more detail about what data we want to keep, is it like a test scenario - test result summary pairs?

Reply:
> I understand it as keeping test results, yes this data can be seen from dashboard but I thought it only shows some windowed data, and at the future we cant see initial results.

yes - it only shows some window (1month? or sth like that).

> So could you give me more detail about what data we want to keep, is it like a test scenario - test result summary pairs?

My personal feeling is that all we need is:
scenario -> [pod startup time percentiles] (just 3 numbers)
mapping.

But let's wait for @msau42 to get her opinion too.

Reply:
Yes I agree I just meant keep a summary of our experiments while we're adjusting all the parameters.  i'm interested in:
* Scenario: max-per-pod, max-per-node
* Important parameters: # pods, # volumes per pod
* Pod startup time percentiles
* Link to one of the test logs? How long do we keep these test run artifacts?

Reply:
Yes - by scenario I meant both what it does and it's params.

Re link to logs - I think we have TTL of 2months.

Reply:
ok, I'll prepare that kind of doc, I think @wojtek-t qps change is another parameter in here.

Reply:
I replaced json file with a yaml file for test summary information